test/exercism/use_cases/attempt_test.rb

```diff
@@ -13,6 +13,25 @@ def python_two
     {'python/two/two.py' => 'CODE'}
   end
 
+  def test_add_comment_to_submission
+    Xapi.stub(:exists?, true) do
+      attempt = Attempt.new(user, Iteration.new(python_two))
+      # REVIEW: we have to call save here because
+      # creating a new attempt creates an invalid submission
+      # once we assign it to the user on save, then we can create a comment.
+      # I'm wondering if it would be better to add a second argument to attempt
+      # that will add a comment to the submission if present on save.
```

Comment:
Yeah, I think that's a good suggestion. All of this code is kind of awful :/ It's still full of old experiments. I feel like I need to apologize every time someone sees it.